extensions/amp-ad/0.1/concurrent-load.js

```diff
@@ -65,10 +75,16 @@ export function incrementLoadingAds(win, opt_loadingPromise) {
     win[LOADING_ADS_WIN_ID_] = 0;
   }
   win[LOADING_ADS_WIN_ID_]++;
+  throttlePromise_ = throttlePromise_ ||
+      new Promise(resolver => throttlePromiseResolver_ = resolver);
   Services.timerFor(win)
       .timeoutPromise(1000, opt_loadingPromise)
       .catch(() => {})
       .then(() => {
-        win[LOADING_ADS_WIN_ID_]--;
+        if (!--win[LOADING_ADS_WIN_ID_]) {
```

Comment:
Nit: I feel that ```if (--win[LOADING_ADS_WIN_ID_]<=0)``` is safer. It _shouldn't_ ever go below zero, but would protect us if that happens as a side-effect of some other mistake.